### PR TITLE
Add Sage 2.1.0 benchmark

### PR DESCRIPTION
**Pull request by ntBre**

### CODE REVIEW

File: submissions/2024-11-07-Sage-2.1.0/yds.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+forcefield: openff-2.1.0.offxml
```
Comment (mattwthompson) [inline submissions/2024-11-07-Sage-2.1.0/yds.yaml:1]:
```suggestion
forcefield: openff_unconstrained-2.1.0.offxml
```
See recent discussion on Slack - assuming you want to use the unconstrained version, that is

Comment (ntBre) [reply submissions/2024-11-07-Sage-2.1.0/yds.yaml:1]: Thanks, I think you're right. I'm planning just to let this one run to make sure I didn't break anything with the last PR (I should probably set up some actual test CI soon) and then I'll open another PR with the unconstrained version.

**Comment by ntBre** [comment]
Just rebased onto master to get the new plotting script.

**Comment by ntBre** [comment]
The record has been published to [Zenodo](https://zenodo.org/records/14053222) with DOI: https://doi.org/10.5281/zenodo.14053221

Please add this to the main README table before merging.

**Pull request merged**